app/Http/helpers.php

```diff
@@ -26,6 +26,22 @@ function transactional_country_code($country_code)
 }
 
 /**
+ * Try to get a value from the config array at the given path, with an optional
+ * fallback path if the desired path is falsy (e.g. empty string or null).
+ *
+ * @param $configPath - Dot notation path for the config array
+ * @param $key - Desired key
+ * @param $fallbackKey - Fallback key, if desired value is falsy
+ */
+function filtered_config($configPath, $key, $fallbackKey)
+{
+    $filteredConfig = array_filter(config($configPath));
+    $key = array_key_exists($key, $filteredConfig) ? $key : $fallbackKey;
+
+    return $filteredConfig[$key];
```

Comment:
Yeah, nothing specifically ensures that here... although at least in the cases I'm thinking of we'd always want to make sure whatever the fallback value was is set, so it's probably okay for it to cause an exception.